src/lib/update-alternatives/model/alternative.rb

```diff
@@ -95,6 +104,26 @@ def empty?
       false
     end
 
+    def choice(new_choice_path)
```

Comment:
I don't like this name. The method is performing an action on the alternative object. The name should reflect that. In this case I would go for `choose` (or even `choose!` since I like to make it clear  by using "!" when a method modifies the object).

Another option is to call the method `value=`